lib/ansible/config/base.yml

```diff
@@ -1390,6 +1390,12 @@ HOST_PATTERN_MISMATCH:
   - {key: host_pattern_mismatch, section: inventory}
   choices: ['warning', 'error', 'ignore']
   version_added: "2.8"
+INSTALLED_CONTENT_ROOTS:
+  name: root paths for loading installed role/plugin content
+  default: ['~/.ansible/content', '/usr/share/ansible/content']
+  type: list
```

Comment:
type paths to match other we do same with? then : separated, its friendlier for env vars

Reply:
fair enough, updated